common.mk

```diff
@@ -24,11 +24,13 @@ ifneq (,$(wildcard ./test/__setup.js))
 	@mocha --recursive --ui tdd \
 	    --require babel-register \
 	    --require babel-polyfill \
-	    --require ./test/__setup
+	    --require ./test/__setup \
+	    test/
```

Comment:
I was getting some strange `unknown option --recursive` errors from `mocha` without this.